3303Elevator/src/app/Config/Config.java

```diff
@@ -0,0 +1,60 @@
+package app.Config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/***
+ * Class used to load configuration files
+ * @author Ben Kittilsen
+ */
+public class Config {
+	
+	/***
+	 * Properties from the loaded configuration file
+	 */
+	private Properties prop;
+	
+	/***
+	 * Loads Properties from a configuration file looks in default 
+	 * configuration path. 
+	 * Default path is "src/app/config"
+	 * 
+	 * @param filename for the file in the default directory
+	 */
+	public Config(String filename){
+		this("src/app/config", filename);
+	}
```

Comment:
weird things can happen when using relative paths on different systems. Dynamically determine the absolute path instead like the below example from MainProgramRunner

System.getProperty("user.dir")+"\\src\\app\\FloorSubsystem\\inputfile.txt";